### ISSUE TITLE
"export 'normalizeURL' was not found in '@nuxt/ufo' [IE]

### ISSUE THREAD

**Issue by andorfermichael** [nuxt/nuxt.js#8458]
<!-- 💚 Thanks for your time to make Nuxt better with your feedbacks 💚

**IMPORTANT** Before reporting a bug:

- Please make sure that you have read through Nuxt documentation: https://nuxtjs.org
- If issue is related to a module please create the issue in corresponding repository
- Ensure using latest version of nuxt dependencies using `yarn upgrade nuxt` or `npm upgrade nuxt`

👍 A properly detailed bug report can save a LOT of time and help fixing issues as soon as possible.
-->

### Versions

- nuxt: v2.14.10
- node: v14.15.0 

<!--### Reproduction-->

<!--
Link to a minimal test case based on one of:
- A fork of https://template.nuxtjs.org
- A GitHub repository that can reproduce the bug
Without a reproduction, it is so hard to address problem :(
-->

<!--<details open>
<summary>Additional Details</summary>
<br> -->
<!-- Attaching `nuxt.config`, dependencies, logs or code snippets would help to find the issue -->
</details>

### What is actually happening?

When running `nuxt` with the latest release 2.14.10, the following warning occurs:

`"export 'normalizeURL' was not found in '@nuxt/ufo'`

#### Command Line Output
```
$ nuxt
ℹ Using Tailwind CSS from ~/assets/css/tailwind.css                                                                                                                                                                      nuxt:tailwindcss 10:30:21
ℹ Merging Tailwind config from ~/tailwind.config.js                                                                                                                                                                      nuxt:tailwindcss 10:30:21
ℹ [HPM] Proxy created: /  -> https://staging.example.com                                                                                                                                                                                 10:30:22
ℹ [HPM] Proxy rewrite rule created: "^/api/" ~> "/spa/"                                                                                                                                                                                   10:30:22

   ╭───────────────────────────────────────╮
   │                                       │
   │   Nuxt @ v2.14.10                     │
   │                                       │
   │   ▸ Environment: development          │
   │   ▸ Rendering:   server-side          │
   │   ▸ Target:      server               │
   │                                       │
   │   Listening: http://localhost:3000/   │
   │                                       │
   ╰───────────────────────────────────────╯

ℹ Preparing project for development                                                                                                                                                                                                       10:30:23
ℹ Initial build may take a while                                                                                                                                                                                                          10:30:23
ℹ Tailwind Colors available on http://localhost:3000/_tailwind                                                                                                                                                           nuxt:tailwindcss 10:30:23
✔ Builder initialized                                                                                                                                                                                                                     10:30:23
✔ Nuxt files generated                                                                                                                                                                                                                    10:30:23

✔ Client
  Compiled successfully in 22.97s

✔ Server
  Compiled successfully in 21.86s


 WARN  Compiled with 1 warnings                                                                                                                                                                                           friendly-errors 10:30:46


 WARN  in ./.nuxt/utils.js                                                                                                                                                                                                friendly-errors 10:30:46

"export 'normalizeURL' was not found in '@nuxt/ufo'                                                                                                                                                                       friendly-errors 10:30:46
                                                                                                                                                                                                                          friendly-errors 10:30:46
ℹ Waiting for file changes                                                                                                                                                                                                                10:30:46
ℹ Memory usage: 319 MB (RSS: 704 MB)                                                                                                                                                                                                      10:30:46
ℹ Listening on: http://localhost:3000/                                                                                                                                                                                                    10:30:46
```

#### Browser Console Output

```
TypeError: Object(...) is not a function
    NuxtJS 2
        getLocation
        _callee2$
    Babel 10
        tryCatch
        invoke
        defineIteratorMethods
        asyncGeneratorStep
        _next
        promise callback*asyncGeneratorStep
        _next
        _asyncToGenerator
        _asyncToGenerator
        _createApp
    NuxtJS 11
        createApp
        <anonymous>
        <anonymous>
        js
        __webpack_require__
        fn
        0
        __webpack_require__
        checkDeferredModules
        webpackJsonpCallback
        <anonymous>
[HMR] bundle 'client' has 2 warnings client.js:195
./.nuxt/utils.js 455:9-21
"export 'normalizeURL' was not found in '@nuxt/ufo'
./.nuxt/router.js 255:11-23
"export 'normalizeURL' was not found in '@nuxt/ufo'

```

Although it is defined as warning, it causes parts of the app not being rendered (correctly).


### What is Expected?

Export should be found, and app should be rendered correctly.

**Comment by andorfermichael** [comment]
This might be related to the recent changes in commit https://github.com/nuxt/nuxt.js/commit/fc51ca333062519d8376262eff93362e3a5739f9

**Comment by snakemastr** [comment]
I also have an issue regarding normalizeUrl in IE11, since version 2.14.10.

![image](https://user-images.githubusercontent.com/18302945/101623062-2d62ae00-3a18-11eb-8219-2413b11fcc55.png)

The text is in Dutch but means "Action is not supported by this object". 

Took me a long time to find as this didnt happen locally, because we never commit package-lock.json (We'll start doing that as of now).

**Comment by pi0** [comment]
Thanks for report. Indeed it is related to recent change. I've noticed this error for IE11 (https://github.com/nuxt/nuxt.js/pull/8430#discussion_r536210643) seems something goes wrong with URL usage.

**Comment by pi0** [comment]
This should be resolved with [v2.14.11](https://github.com/nuxt/nuxt.js/releases/tag/v2.14.11).

**Comment by andorfermichael** [comment]
For me the error still persists in the command line as well as in Firefox 83.0 and Google Chrome 87.0.4280.88.

**Comment by pi0** [comment]
@andorfermichael Can you please try cleaning node_modules/lock file or possibly sharing project to investigate? issue i was referring to resolved was about ie support

**Comment by amritk** [comment]
can confirm it still exists on chrome after clearing node_modules

![image](https://user-images.githubusercontent.com/2039539/102286040-4c67b100-3eec-11eb-8cb9-5d3016581f48.png)
![image](https://user-images.githubusercontent.com/2039539/102286049-52f62880-3eec-11eb-9df4-42bf5eef4f1a.png)

Although I do see normalizeURL exported from '@nuxt/ufo`

**Comment by pi0** [comment]
Hi @amritk Can you please share config and if using programmatic usage? Any way ro reproduce can help to fix issue 😊

**Comment by pi0** [comment]
@amritk Would you please try [v2.14.12](https://github.com/nuxt/nuxt.js/releases/tag/v2.14.12)?

**Comment by andorfermichael** [comment]
> @amritk Would you please try [v2.14.12](https://github.com/nuxt/nuxt.js/releases/tag/v2.14.12)?

v2.14.2 causes another error #8498

**Comment by andorfermichael** [comment]
With the hotfix mentioned above and the fix for #8498 it works now.

Thank you for the quick fix.

**Comment by amritk** [comment]
@pi0 works great now thanks!
